s3cortxutils/s3msgbus/s3msgbus/cortx_s3_msgbus.py

```diff
@@ -110,3 +110,14 @@ def ack(self):
                 exception, traceback.format_exc())
             return False, msg
         return True, None
+
+    def count(self, consumer_group):
+        """Get the count of unread messages."""
+        count = 0
+        try:
+            count = self._producer.get_unread_count(consumer_group)
+        except Exception as exception:
+            msg = ("msg_bus unread_count except:%s %s") % (
+                exception, traceback.format_exc())
+            return msg
```

Comment:
if you return msg how will you return count. same function should not return different types.
This function should only return count. count should be 0 in case of failure and count in case of success.

Reply:
done